sortfilterproxymodel.qbs

```diff
@@ -46,6 +47,7 @@ Product
             return defines
         }
 
+        cpp.linkerFlags: "--no-as-needed"
```

Comment:
@saidinesh5 @johnbivens @BaCaRoZzo @MarcelBlanck 
Do anyone see a problem when we set the linkerflag "--no-as-needed" in the export block from this lib?